vnext/Microsoft.ReactNative/Fabric/Composition/TextInput/WindowsTextInputComponentView.cpp

```diff
@@ -639,6 +638,14 @@ void WindowsTextInputComponentView::updateProps(
     }
   }
 
+  if (oldTextInputProps.placeholder != newTextInputProps.placeholder) {
+    m_placeholderText = newTextInputProps.placeholder;
```

Comment:
Actually we already store the props, so you should be able to just use m_props.placeholder where you are currently using m_placeholderText.

The only thing you would need to do if the placeholder or placeholderTextColor change, is set a dirty flag so that we redraw in those cases.  (Currently we dont redraw if either of these properties change).

We already have a dirty flag (m_needsRedraw).  So I think you could probably just set that, then add a if(m_needsRedraw) { DrawText() } at the end of finalizeUpdates

Reply:
ahhh that makes sense and removes the extra storage, thanks!!